webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/RudderDit.scala

```diff
@@ -309,6 +352,32 @@ class RudderDit(val BASE_DN:DN) extends AbstractDit {
     }
   }
 
+  object RULETARGET {
+
+      def ruleTargetDN(targetId:String) : RDN = new RDN(A_ACTIVE_TECHNIQUE_UUID, targetId)
```

Comment:
What should I put instead of `A_ACTIVE_TECHNIQUE_UUID`

Reply:
`RDN` is the attribute used as identifier for entry. So for node, it's `nodeid` (`A_NODE_UUID`). 
Ldap entries are located by their full name, the path from root of tree to them, that we call a distinguished name (dn). 
rdn is the last part of the path, so that `entry DN = entry RDN, parent DN` (recursively until root)